test/apiv2/20-containers.at

```diff
@@ -113,18 +113,18 @@ t GET libpod/containers/json?last=1 200 \
 cid=$(jq -r '.[0].Id' <<<"$output")
 
 if root; then
-    t GET libpod/containers/stats?containers='[$cid]' 200
+    t GET "libpod/containers/stats?containers=$cid&stream=false" 200
 else
     if have_cgroupsv2; then
-        t GET libpod/containers/stats?containers='[$cid]' 200
+        t GET "libpod/containers/stats?containers=$cid&stream=false" 200
     else
-        t GET libpod/containers/stats?containers='[$cid]' 409
+        t GET "libpod/containers/stats?containers=$cid&stream=false" 200
```

Comment:
I guess this still returns 409 for cgroup v1, we just don't test cgroupv1 in CI for the pai tests so we don't notice I guess and we in the process of dropping all cgroupv1 testing so I guess you might as well just drop all the conditionals